src/test/java/vehicle/PostVehicleTechnicalRecordsSpecialistTesting.java

```diff
@@ -0,0 +1,204 @@
+package vehicle;
+
+
+import data.GenericData;
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import steps.VehicleTechnicalRecordsSteps;
+import util.JsonPathAlteration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.http.HttpStatus;
+
+
+@RunWith(SerenityRunner.class)
+public class PostVehicleTechnicalRecordsSpecialistTesting {
```

Comment:
Maybe all the positive scenarios for post and put on tech records we could move them in the existing classes PostVehicleTechnicalRecords and PutVehicleTechnicalRecords, since for the TRL, HGV and PSV the positive scenarios we kept them together in the same class.

Reply:
Changes done.